openshift/server.dc.yml

```
@@ -119,24 +119,18 @@ objects:
                       key: sa-password
                       name: ${APP_NAME}-keycloak
                 - name: MONGO_HOST
-                  value: ${APP_NAME}-mongodb
+                  value: ${APP_NAME}-mongo-headless
                 - name: MONGO_PORT
                   value: '27017'
                 - name: MONGO_DB
-                  valueFrom:
-                    secretKeyRef:
-                      key: database
-                      name: ${APP_NAME}-mongodb
+                  value: hcap
```

Is it worth using the app name variable here?

```suggestion
                  value: ${APP_NAME}
```